deploy/helm/lunettes/templates/grafanadi/grafanadi-deploy.yaml

```diff
@@ -20,9 +20,25 @@ spec:
           - 0
       containers:
       - command:
-        - ./grafanadi
-        - --config-file=/app/config-file.yaml
-        - --enable-kube-client={{ .Values.grafanadiInitKubeClient }}
+        - /bin/sh
+        - -c
+        - |
+          echo "Waiting for elastic search cluster to start..."
+          status_code="000X"
+          while true;
+          do
+            if [ ${status_code} = "200X" ];
+            then
+              echo "Elastic search cluster start successfully!"
+              break
+            fi
+            status_code=`curl -s --connect-timeout 3 --max-time 5 \
+            http://es-cluster-svc.lunettes:9200 \
```

Comment:
use `http://es-cluster-svc.{{ .Values.namespace }}:9200` instead of `http://es-cluster-svc.lunettes:9200`, because namespace maybe different